node/src/components/contract_runtime/utils.rs

```diff
@@ -112,7 +112,9 @@ pub(super) async fn exec_or_requeue<REv>(
     }
 
     let maybe_next_era_gas_price = if is_era_end && executable_block.next_era_gas_price.is_none() {
-        let block_max_standard_count = chainspec.transaction_config.block_max_standard_count;
+        let block_max_large_count = chainspec.transaction_config.block_max_large_count;
+        let block_max_medium_count = chainspec.transaction_config.block_max_large_count;
+        let block_max_small_count = chainspec.transaction_config.block_max_large_count;
```

Comment:
both med & small are being set to block_max_large_count;

also, the determination of these settings is meant to be a runtime determination from the chainspec file multi-array, not static fields.